### PR TITLE
21535 - Update url in eft refund email

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:*
https://github.com/bcgov/entity/issues/21535

*Description of changes:*
1. Update url in eft refund email
2. Re-create table eft_refund_email_list with auto-increment id

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

File: pay-api/migrations/versions/2024_07_12_f921d5e32835_add_expense_authority_email.py [hunk 1]
```diff
@@ -22,6 +22,7 @@ def upgrade():
                     sa.Column('first_name', sa.String(25), nullable=True),
                     sa.Column('last_name', sa.String(25), nullable=True),
                     sa.Column('email', sa.String(25), nullable=False),
+                    sa.PrimaryKeyConstraint('id')
```
Comment (ochiu) [inline pay-api/migrations/versions/2024_07_12_f921d5e32835_add_expense_authority_email.py:25]: This migration would have run already. I think you will need a new one to make changes to add a PK constraint to id.

Comment (Jxio) [reply pay-api/migrations/versions/2024_07_12_f921d5e32835_add_expense_authority_email.py]: fixed

File: pay-api/migrations/versions/2024_07_22_f9c15c7f29f5_.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+
+def upgrade():
+    op.drop_table('eft_refund_email_list')
+
+    op.execute(CreateSequence(Sequence('eft_refund_email_list_id_seq')))
+
+    op.create_table('eft_refund_email_list',
+        sa.Column('id', sa.Integer(), 
+                  sa.Sequence('eft_refund_email_list_id_seq'),
```
Comment (ochiu) [inline pay-api/migrations/versions/2024_07_22_f9c15c7f29f5_.py:27]: If you are going to use a sequence, the model should probably refer to it as well.

**Review by ochiu — APPROVED** [review]

File: pay-api/migrations/versions/2024_07_22_f9c15c7f29f5_.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+# revision identifiers, used by Alembic.
+revision = 'f9c15c7f29f5'
+down_revision = 'fb59bf68146d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.drop_table('eft_refund_email_list')
```
Comment (seeker25) [inline pay-api/migrations/versions/2024_07_22_f9c15c7f29f5_.py:21]: probably not the best practice, if someone was using this table? but should be fine

**Review by seeker25 — APPROVED** [review]

File: pay-api/migrations/versions/2024_07_22_f9c15c7f29f5_.py (hunk 3 above)
Comment (Jxio) [reply pay-api/migrations/versions/2024_07_22_f9c15c7f29f5_.py:21]: this table is not deployed yet, so it's empty

**Pull request merged**